package.json

```diff
@@ -267,7 +267,8 @@
         "visible": true,
         "subPermissions": [
           "ui-invoice.pay",
-          "finance.fiscal-years.collection.get"
+          "finance.fiscal-years.collection.get",
+          "ui-finance.fiscal-year.view"
```

Comment:
why did you include `ui-finance.fiscal-year.view` to this permission? if user can pay different against FY, it doesn't mean that use should access Finance application and FY list

all you need is to add `finance.fiscal-years.collection.get` permissions for permission set that is used to display invoice list

Reply:
@alisher-epam 
> all you need is to add finance.fiscal-years.collection.get permissions for permission set that is used to display invoice list